docs/organizations/managing-security-and-risk.md

```diff
@@ -0,0 +1,108 @@
+# Managing security and risk
+
+!!! info "This is a beta feature"
+    This is a new Codacy feature and <span class="skip-vale">we're</span> continuing to improve it.
+
+To <span class="skip-vale">quickly</span> identify, prioritize, track, and address security vulnerabilities, open the **Security and risk management** page from the organization-level navigation menu. On this page, you can monitor and act on time-bound, prioritized action items that Codacy creates automatically when finding Security issues:
+
+-   Codacy opens a new item when it finds a new Security issue on the default branch of a repository.
+
+    [Every new item is assigned a priority](#item-priorities-and-deadlines) based on the severity of the underlying issue. Each priority also defines a deadline to fix the underlying issue.
+
+    [Every new item is assigned a status](#item-statuses) based on the number of days left before the deadline to fix the underlying issue. As deadlines are approached, met, or missed, items transition through different statuses.
+
+-   Codacy closes an item when it stops detecting the underlying issue, when [the underlying issue is ignored](../repositories/issues.md#ignoring-and-managing-issues), or when the [tool that found the underlying issue is disabled](../repositories-configure/configuring-code-patterns.md).
+
+For example, Codacy automatically opens an item with Critical priority upon finding a new Security issue of Critical severity on the default branch of a repository. If the Security issue is fixed before the deadline, the status of the item changes to Closed on time. If the issue isn't fixed before the deadline, the status of the item changes to Overdue.
+
+<!-- TODO reword -->
```

Comment:
Definitely a TODO here! 👍 
Probably the Items list will require a new page, but we need to understand how the UI will navigate from the Dashboard to the list, and what will be the use case.

Reply:
I'd keep this on a single page for now. The feature may [spin off into a separate product](https://codacy.slite.com/app/docs/bRGd2kKUijL5BX#457ffc51) and will likely evolve to have its own section.

Reply:
Yes, let's keep this self-contained for now. :ok_hand: